shaderc-sys/build/build.rs

```diff
@@ -33,6 +36,9 @@ fn build_shaderc(shaderc_dir: &PathBuf, use_ninja: bool) -> PathBuf {
         .define("SPIRV_WERROR", "OFF")
         .define("SHADERC_SKIP_TESTS", "ON")
         .define("CMAKE_INSTALL_LIBDIR", "lib");
+    if target_os == "windows" && target_env == "gnu" {
+        config.define("_GLIBCXX_USE_CXX11_ABI", "0");
```

Comment:
Thanks for the patch. I'm not very familiar with the MinGW environment. So MinGW actually uses the old ABI? Is this safe if they later change to the new ABI? Is there a way to detect which ABI to use?

It would be nice to paste the link here as a comment so folks can easily discover the reason of this definition. :)